index.html

```diff
@@ -1,176 +1,230 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
     <title>GU ACM</title>
-    <link rel="stylesheet" type="text/css" href="./CSS/navbar.css">
+
+    <!-- External CSS -->
     <link rel="stylesheet" type="text/css" href="./CSS/about.css">
+    <link rel="stylesheet" type="text/css" href="./CSS/calendar.css">
+    <link rel="stylesheet" type="text/css" href="./CSS/events.css">
+    <link rel="stylesheet" href="./CSS/evo-calendar.min.css">
+    <link rel="stylesheet" href="./CSS/evo-calendar.midnight-blue.min.css">
+    <link rel="stylesheet" type="text/css" href="./CSS/footer.css">
+    <link rel="stylesheet" type="text/css" href="./CSS/landing_page.css">
+    <link rel="stylesheet" type="text/css" href="./CSS/navbar.css">
     <link rel="stylesheet" type="text/css" href="./CSS/officers.css">
```

Comment:
Should we simplify this? Seems like we shouldn't have so many links like this. or we simplify the css somehow.

@JT2M0L3Y @NisaMeshal @Colexeco

Reply:
We are planning to condense this into one CSS file.